lang/cs.json

```diff
@@ -67,19 +67,19 @@
   "Depressed": "Propadlý",
   "Uniform": "Rovnoměrný",
   "Dropshadow": "Stínovaný",
-  "Font Family": "Titulky",
-  "Proportional Sans-Serif": "Proporcionální Sans-Serif",
-  "Monospace Sans-Serif": "Monospace Sans-Serif",
-  "Proportional Serif": "Proporcionální Serif",
-  "Monospace Serif": "Monospace Serif",
+  "Font Family": "Rodina písma",
+  "Proportional Sans-Serif": "Proporcionální bezpatkové",
+  "Monospace Sans-Serif": "Monospace bezpatkové",
+  "Proportional Serif": "Proporcionální patkové",
+  "Monospace Serif": "Monospace patkové",
   "Casual": "Hravé",
   "Script": "Ručně psané",
-  "Small Caps": "Velkými písmeny",
+  "Small Caps": "Malé kapitálky",
   "Reset": "Obnovit",
   "restore all settings to the default values": "Vrátit nastavení do výchozího stavu",
   "Done": "Hotovo",
-  "Caption Settings Dialog": "Okno s nastevením !caption!",
-  "Beginning of dialog window. Escape will cancel and close the window.": "Začátek dialogového okna. Klávesa Escape okno zavře.",
+  "Caption Settings Dialog": "Okno s nastavením titulků",
+  "Beginning of dialog window. Escape will cancel and close the window.": "Začátek dialogového okna. Klávesa Esc okno zavře.",
   "End of dialog window.": "Konec dialogového okna.",
-  "{1} is loading.": "{1} se načítá."
+  "{1} is loading.": "{1} se načítá.",
```

Comment:
Trailing `,` is invalid JSON.

Reply:
Why is that? It would be better for diffs, no?

Reply:
It's just how JSON is. With the trailing comma, `JSON.parse()` will throw an error. It comes from the bad old days when certain JS engines would choke on trailing commas.